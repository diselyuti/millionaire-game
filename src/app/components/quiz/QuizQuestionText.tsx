import React from 'react';
import styles from '@/app/styles/components/quiz/quiz-question.module.scss';

function QuizQuestionText({ text }: { text: string }) {
  return <h2 className={styles.question}>{text}</h2>;
}

export default QuizQuestionText;
