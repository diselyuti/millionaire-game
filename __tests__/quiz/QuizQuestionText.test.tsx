import React from 'react';
import '@testing-library/jest-dom';
import QuizQuestionText from "@/app/components/quiz/QuizQuestionText.tsx";
import {render} from "@testing-library/react";

describe('QuizQuestionText', () => {
  test('should render text', () => {
    const { container } = render(<QuizQuestionText text="Question" />);
    expect(container).toHaveTextContent('Question');
  });

  test('should render QuizQuestionText', () => {
    const { container } = render(<QuizQuestionText text="Question" />);
    expect(container).toMatchSnapshot();
  });
});
