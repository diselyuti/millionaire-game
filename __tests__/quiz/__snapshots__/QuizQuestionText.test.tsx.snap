// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizQuestionText should render QuizQuestionText 1`] = `
<div>
  <h2
    class="question"
  >
    Question
  </h2>
</div>
`;
